src/libcore/num/mod.rs

```diff
@@ -1979,10 +1979,10 @@ When starting from a slice rather than an array, fallible conversion APIs can be
 ```
 use std::convert::TryInto;
 
-fn read_be_", stringify!($SelfT), "(input: &mut &[u8]) -> ", stringify!($SelfT), " {
+fn read_le_", stringify!($SelfT), "(input: &mut &[u8]) -> ", stringify!($SelfT), " {
     let (int_bytes, rest) = input.split_at(std::mem::size_of::<", stringify!($SelfT), ">());
     *input = rest;
-    ", stringify!($SelfT), "::from_be_bytes(int_bytes.try_into().unwrap())
+    ", stringify!($SelfT), "::from_le_bytes(int_bytes.try_into().unwrap())
```

Comment:
I see there's a `le_bytes` in 

https://github.com/rust-lang/rust/blob/0561060d23b804dde8e7e3bd85df4b1fd043747d/src/libcore/num/mod.rs#L198-L201

Should this test be using that?

Reply:
Updated the PR.